src/components/MultipleChoice/BaseMultipleChoice.tsx

```diff
@@ -39,6 +39,8 @@ export const BaseMultipleChoice: FC<BaseMultipleChoiceProps> = ({
 
 	const [currentFocus, setCurrentFocus] = useState(-1)
 
+	const keysPressedMap: KeysPressedMap = useMemo(() => ({}), [])
```

Comment:
Why are we using `useMemo` here?  If you are worried about constant re-renders from updating the map, I think it would be more appropriate to do `useRef` since updating refs don't cause the components to re-render.

```
const keysPressedMapRef = useRef<KeysPressedMap>({})

// onKeyDown
keysPressedMapRef.current[e.key] = true

// To pass as props
keysPressedMap={keysPressedMapRef.current}
```